### PR TITLE
[fabric] #1314 Fix and test existing molecule scenarios

### PR DESCRIPTION
**Pull request by TonyRowntree**
Signed-off-by: TonyRowntree <33454202+TonyRowntree@users.noreply.github.com>

**Changelog**
- Fix molecule scenario Crypto, Default and Pre-Vault-Setup

**Reviewed by**
@sownak 

 

**Linked issue**
#1314

### CODE REVIEW

File: platforms/hyperledger-fabric/configuration/molecule/crypto/verify.yml [hunk 1]
```diff
@@ -317,16 +317,16 @@
 
 #####################################################################################
   #This task checks if genesis block is created or not
-  - name: Check if the genesis block is created or not
-    stat:
-      path: "./build/channel-artifacts/genesis.block.base64"
-    register: genesis_check
+  #- name: Check if the genesis block is created or not
```
Comment (jagpreetsinghsasan) [inline platforms/hyperledger-fabric/configuration/molecule/crypto/verify.yml:320]: You don't need to comment the code. As this piece of code is irrelevant, you can just remove this piece of code.

File: platforms/hyperledger-fabric/configuration/molecule/crypto/verify.yml [hunk 2]
```diff
@@ -317,16 +317,16 @@
...
+    #stat:
+      #path: "./build/channel-artifacts/genesis.block.base64"
+    #register: genesis_check
   
 #####################################################################################
   #This task assert if  genesis block is created or not
-  - name: Assert if the genesis block is created or not
-    assert:
-      that: genesis_check.stat.exists
+  #- name: Assert if the genesis block is created or not
```
Comment (jagpreetsinghsasan) [inline platforms/hyperledger-fabric/configuration/molecule/crypto/verify.yml:327]: You don't need to comment the code. As this piece of code is irrelevant, you can just remove this piece of code.

**Review by jagpreetsinghsasan — CHANGES_REQUESTED** [review]

**Comment by jagpreetsinghsasan** [comment]
The molecule test failed. Have a look, as in why it failed.

**Comment by sownak** [comment]
@TonyRowntree I have made some changes so you will need to update your branch with the latest changes from develop

**Comment by TonyRowntree** [comment]
Have made the changes and fixed issue.

Please review @sownak @jagpreetsinghsasan

**Review by sownak — APPROVED** [review]

**Pull request merged**